src/app/check-in/[slug]/page.tsx

```
@@ -355,15 +359,32 @@ export default function CheckInDetailsPage() {
+                          );
+                        }}
                         className="bg-blue-500 hover:bg-blue-600"
+                        disabled={isNavigating}
                       >
-                        <CalendarPlus className="w-5 h-5 mr-2" />
-                        Reservar Ahora
+                        {isNavigating ? (
+                          <>
+                            <motion.div
+                              animate={{ rotate: 360 }}
+                              transition={{ duration: 1, repeat: Infinity, ease: "linear" }}
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)